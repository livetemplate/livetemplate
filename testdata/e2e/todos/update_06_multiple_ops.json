{
  "10": "2023-01-01 11:30:00",
  "2": "11",
  "6": "2",
  "7": "2",
  "8": "50%",
  "9": [
    [
      "r",
      "todo-5"
    ],
    [
      "r",
      "todo-3"
    ],
    [
      "u",
      "todo-4",
      {
        "1": "completed",
        "5": "✓"
      }
    ],
    [
      "u",
      "todo-1",
      {
        "3": "#1"
      }
    ],
    [
      "i",
      "todo-1",
      "after",
      [
        {
          "0": "todo-6",
          "1": "",
          "2": "todo-6",
          "3": "#2",
          "4": "Deploy to production",
          "5": "○",
          "6": "(Priority: Critical)"
        },
        {
          "0": "todo-7",
          "1": "",
          "2": "todo-7",
          "3": "#3",
          "4": "Monitor performance",
          "5": "○",
          "6": "(Priority: Medium)"
        }
      ]
    ]
  ]
}
