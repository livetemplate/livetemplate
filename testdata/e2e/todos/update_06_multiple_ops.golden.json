{
  "1": "11",
  "10": "2023-01-01 11:30:00",
  "5": "2",
  "6": "2",
  "7": "50%",
  "8": "\u003cdiv class=\"todo-list\"\u003e\n                    \n                    \u003cdiv class=\"todo-item completed\" data-key=\"todo-4\"\u003e\n                        \u003cstrong\u003e#0:\u003c/strong\u003e\n                        Setup development environment\n                        ✓\n                         (Priority: High)\n                    \u003c/div\u003e\n                    \n                    \u003cdiv class=\"todo-item completed\" data-key=\"todo-1\"\u003e\n                        \u003cstrong\u003e#1:\u003c/strong\u003e\n                        Learn Go templates\n                        ✓\n                         (Priority: High)\n                    \u003c/div\u003e\n                    \n                    \u003cdiv class=\"todo-item \" data-key=\"todo-6\"\u003e\n                        \u003cstrong\u003e#2:\u003c/strong\u003e\n                        Deploy to production\n                        ○\n                         (Priority: Critical)\n                    \u003c/div\u003e\n                    \n                    \u003cdiv class=\"todo-item \" data-key=\"todo-7\"\u003e\n                        \u003cstrong\u003e#3:\u003c/strong\u003e\n                        Monitor performance\n                        ○\n                         (Priority: Medium)\n                    \u003c/div\u003e\n                    \n                \u003c/div\u003e",
  "9": [
    [
      "r",
      "todo-3"
    ],
    [
      "r",
      "todo-5"
    ],
    [
      "i",
      "todo-1",
      "after",
      [
        {
          "0": "todo-6"
        },
        {
          "0": "todo-7"
        }
      ]
    ]
  ]
}